ambry-router/src/test/java/com.github.ambry.router/InMemoryRouter.java

```diff
@@ -376,6 +397,23 @@ private ByteBuffer readBlob(ReadableStreamChannel postContent) throws Interrupte
     }
     return ByteBuffer.wrap(blobDataStream.toByteArray());
   }
+
+  /**
+   * Choose a random {@link PartitionId} for putting the current chunk and return it. This code is copied from
+   * {@code PutOperation#getPartitionForPut}.
+   * @param partitionIdsToExclude the list of {@link PartitionId}s that should be excluded from consideration.
+   * @return the chosen {@link PartitionId}
+   * @throws RouterException
+   */
+  private PartitionId getPartitionForPut(List<PartitionId> partitionIdsToExclude) throws RouterException {
```

Comment:
why do you need the `partitionIdsToExclude` variable? The only user sends in an empty list anyway